backend/packages/Upgrade/src/api/controllers/FeatureFlagController.ts

```diff
@@ -338,6 +339,44 @@ export class FeatureFlagsController {
     return this.featureFlagService.updateState(flag.flagId, flag.status);
   }
 
+  /**
+   * @swagger
+   * /flags/filterMode:
+   *    post:
+   *       description: Update Feature flag Filter Mode
+   *       consumes:
+   *         - application/json
+   *       parameters:
+   *         - in: body
+   *           name: updateFilterMode
+   *           description: Updating the featur flag's filter mode
+   *           schema:
+   *             type: object
+   *             required:
+   *              - flagId
+   *              - filterMode
+   *             properties:
+   *              flagId:
+   *                type: string
+   *              filterMode:
+   *                type: string
+   *                enum: [includeAll, excludeAll]
+   *       tags:
+   *         - Feature Flags
+   *       produces:
+   *         - application/json
+   *       responses:
+   *          '200':
+   *            description: Feature flag filterMode is updated
+   */
+  @Post('/filterMode')
```

Comment:
I think it is more like a Path than Post

Reply:
I'm not sure I understand; the path is '/filterMode'.

Reply:
i think he meant PUT

Reply:
Or maybe PATCH

Reply:
Either way, I think it should match what '/status' is, which is POST, and there's already front-end stuff that depends on that, I think.

Reply:
ah maybe patch. if you were to change POST to some other verb and wanted to do it for status as well, the change would just be a one word update data services file, easy peasy https://github.com/CarnegieLearningWeb/UpGrade/blob/dev/frontend/projects/upgrade/src/app/core/feature-flags/feature-flags.data.service.ts#L32

Reply:
Ok, I'll do that. PATCH is indeed probably more appropriate.